.circleci/src/jobs/build_android.yml

```diff
@@ -31,19 +31,24 @@ steps:
       name: "[FL] << parameters.build_config_name >> build"
       command: ./gradlew clean && bundle exec fastlane build version_name:${NEW_VERSION_NAME} version_code:${NEW_VERSION_CODE} build_config_name:<< parameters.build_config_name >>
       working_directory: native/android
+  - run:
+      name: Prepare Sourcemaps
+      command: |
+        mkdir -p sourcemaps/<< parameters.build_config_name >>
+        cp native/android/app/build/generated/sourcemaps/react/release/index.android.bundle.map sourcemaps/<< parameters.build_config_name >>/
```

Comment:
before we used `move` for the sourcemaps. What was your resoning to use `copy` instead?

Reply:
not really a difference, but changing the build directory has the potential to go wrong.